.gitmodules

```diff
@@ -8,8 +8,8 @@
 	url = https://review.coreboot.org/nvidia-cbootimage.git
 [submodule "vboot"]
 	path = 3rdparty/vboot
-	url = https://review.coreboot.org/vboot.git
-	branch = main
+	url = https://github.com/Dasharo/vboot
+	branch = master
```

Comment:
Why not `dasharo`? Also, who is `miczyg1395` and why doesn't he sign his commits?

https://github.com/Dasharo/vboot/commit/22134690d7ced7b2ea824b71b597bb73586d99c6
> [miczyg1395](https://github.com/Dasharo/vboot/commits?author=miczyg1395) authored and [miczyg1](https://github.com/Dasharo/vboot/commits?author=miczyg1) committed

Reply:
miczyg1395 is my private account. I use this account to contribute to vboot as agreed with management. I have simply cherry-picked the commit from vboot gerrit using my work account.

`miczyg1395` does not sign commits as I do not use this acc actively and I do not have GPG key bound to this acc.

Reply:
I do not use dasharo branch, because it is too old. I have simply switched to upstream vboot with the patch I have uploaded for review and pulled from gerrit to our fork.